service_configuration_lib/spark_config.py

```diff
@@ -1043,14 +943,23 @@ def _convert_user_spark_opts_value_to_str(user_spark_opts: Mapping[str, Any]) ->
     return output
 
 
-def compute_approx_hourly_cost_dollars(spark_conf, paasta_cluster, paasta_pool):
+def update_spark_srv_configs(spark_conf: MutableMapping[str, str]):
+    additional_conf = {k: v for k, v in mandatory_default_spark_srv_conf.items() if k not in spark_conf}
+    spark_conf.update(additional_conf)
+
+
+def compute_approx_hourly_cost_dollars(
+        spark_conf: Mapping[str, str],
```

Comment:
Nit: This looks like double the indentation. 
Was it auto-added by pre-commit hook?